### ISSUE TITLE
develop WSGI server support

### ISSUE THREAD

**Issue by GlenDC** [plabayo/rama#343]
Deliverables:

- crate: `rama-wsgi`
- example + test
- documentation

Low priority item that is tackled by me, only after 0.3 is delivered,
as until then it is too much of a fun distraction.

**Comment by GlenDC** [comment]
Dropping this in favour of #612 .
